### PR TITLE
Add workflow for snap upload and release

### PR DESCRIPTION
**Pull request by tpsilva**

### CODE REVIEW

**Comment by tpsilva** [comment]
@dosaboy for this to work, you'll need to add your snapcraft token to a repository secred called STORE_LOGIN. Descriptions on how to do this can be found at: https://github.com/snapcore/action-publish#store-login

File: .github/workflows/publish.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - uses: snapcore/action-build@v1
+      id: build
+    - uses: snapcore/action-publish@v1
+      env:
+        SNAPCRAFT_STORE_CREDENTIALS: ${{ secrets.STORE_LOGIN }}
+      with:
+        snap: ${{ steps.build.outputs.snap }}
+        release: stable
```
Comment (dosaboy) [inline .github/workflows/publish.yaml:20]: can we publish to edge instead of stable so that we get a chance to test a build before it gets released?

Comment (tpsilva) [reply .github/workflows/publish.yaml]: Done

**Comment by nicolasbock** [comment]
Why is a separate action necessary to publish to snapcraft @tpsilva ? If the repository is connected to the snap it will build automatically every time there is a new commit on `main`. What am I missing?

**Pull request merged**